tests/Kernel/MenuLocalTasksTest.php

```diff
@@ -36,6 +37,16 @@ public function testMenuLocalTasks(): void {
             'url' => Url::fromUri('http://www.inactive.com'),
           ],
           '#active' => FALSE,
+          '#weight' => -10,
+        ],
+        'link3.link' => [
+          '#theme' => 'menu_local_task',
+          '#link' => [
+            'title' => 'Middle link',
```

Comment:
If you named this one Middle link, add to the other ones as well "First" and "Second" to the name to indicate their extra purpose in the list.